statify.php

```diff
@@ -53,6 +53,8 @@
 	)
 );
 
+/* Composer Autpload */
+require __DIR__ . '/vendor/autoload.php';
 
 /* Autoload */
 spl_autoload_register( 'statify_autoload' );
```

Comment:
If we now use the Composer autoloader adding this feature, it might make sense to remove our own autoloader. But this could be done in another PR once this is merged.